src/components/exercise-table/Exercise-table.tsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+
+// Define the Set type
+interface Set {
+  reps: number;
+  weight: number;
+}
+
+// Define the Exercise type
+interface ExerciseEntry extends Exercise { 
+  sets: Set[];
+}
+  id: number;
+  name: string;
+  label: string;
+  bodyPart: string;
+  sets: Set[];
+}
```

Comment:
We can delete this part. 
FYI when using the `extends` keyword to extend an interface, all of the members of the interface you are extending are already included. You can just simply add the new members...(in the case it's `sets`)